src/app/find/mapPage/filters/FiltersModal.jsx

```diff
@@ -51,6 +52,11 @@ class FiltersModal extends Component {
             onChange={this.setFilterValues}
           />
         </div>
+        <div className="d-flex justify-content-around filterSubmitBtn">
+          <Button secondary fluid onClick={this.submit} >
+            <span>Apply filters</span>
```

Comment:
Not that it hurts, but does the `span` do anything? (I mean, as opposed to just putting the text as the direct child of the Button.)

Reply:
Fixed!